Sources/Sentry/include/SentryTraceContext.h

```diff
@@ -52,6 +52,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nullable, nonatomic, strong) NSString *sampleRate;
 
+/**
+ * Value indicating whether the trace was sampled.
+ */
+@property (nullable, nonatomic, strong) NSString *sampled;
```

Comment:
These properties should probably be `readonly` and/or `copy` if they're only settable via the init method